apps/src/templates/CodeWorkspace.jsx

```diff
@@ -210,7 +236,10 @@ class CodeWorkspace extends React.Component {
             ref={codeTextbox => (this.codeTextbox = codeTextbox)}
             id="codeTextbox"
             className={this.props.pinWorkspaceToBottom ? 'pin_bottom' : ''}
-          />
+            canupdate={'yes'}
+          >
+            {this.props.displayWorkspaceAlert && this.renderWorkspaceAlert()}
```

Comment:
Should this also be prepended by `props.editCode &&` so we don't display the alert twice?

Reply:
If you look at line 234, it is.

Reply:
Ah got it - thanks for clarifying!